fetchit_challenge/worlds/nec0115a_arena.world

```diff
@@ -22,53 +22,34 @@
     <include>
       <uri>model://rarelab_long_table</uri>
       <name>caddy_table</name>
-      <pose>0.710161 -0.806812 0.0 0 0 -3.141585</pose>
+      <pose>0.710161 -0.915409 0.0 0 0 -3.141585</pose>
     </include>
 
 
     <include>
       <uri>model://caddy_green</uri>
       <name>caddy_green1</name>
-      <pose>-0.032536 -0.942377 0.646347 0 0 0</pose>
+      <pose>0.168355 -1.06637 0.775 0 0 0</pose>
     </include>
 
     <include>
       <uri>model://caddy_red</uri>
       <name>caddy_red1</name>
-      <pose>-0.311593 -0.942377 0.646347 0 0 0</pose>
+      <pose>-0.113059 -1.06637 0.775 0 0 0</pose>
     </include>
 
     <include>
       <uri>model://caddy_yellow</uri>
       <name>caddy_yellow1</name>
-      <pose>-0.604281 -0.942377 0.646347 0 0 0</pose>
+      <pose>-0.414898 -1.06637 0.775 0 0 0</pose>
     </include>
 
     <include>
       <uri>model://gearbox_bottom_grey</uri>
       <name>gearbox_bottom_grey1</name>
-      <pose>-0.424903 0.854808 0.626969 0 0 3.105351</pose>
+      <pose>-0.134223 0.990266 0.734993 0 0 -0.085</pose>
     </include>
 
-    <include>
-      <uri>model://gearbox_bottom_grey</uri>
-      <name>gearbox_bottom_grey2</name>
-      <pose>-0.229921 1.035252 0.655835 -1.570796 0 2.317278</pose>
-    </include>
-
-    <include>
-      <uri>model://gearbox_bottom_grey</uri>
-      <name>gearbox_bottom_grey3</name>
-      <pose>-0.329409 1.222056 0.690921 3.141546 0 -2.437420</pose>
-    </include>
-
-    <include>
-      <uri>model://gearbox_bottom_grey</uri>
-      <name>gearbox_bottom_grey4</name>
-      <pose>-0.482731 1.025975 0.635592 -1.570796 0 -2.472675</pose>
-    </include>
-
-
```

Comment:
Can you put back those gearbox bottoms (with working poses) for testing purposes? We can use https://github.com/TheRARELab/fetch_gazebo/blob/gazebo9/fetchit_challenge/worlds/nec0115a_arena_virtual_virtual.world for the one-gearbox-bottom case. What do you think?

Reply:
I.e., we will replicate it here to where we put the objects on the physical table, but just remove 1 (which is the one in the virtual/virtual world)

Reply:
okay, I put them back with the correct poses